android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt

```diff
@@ -123,6 +123,8 @@ private val textFieldBorderColorDark = Color((0xFFE5E5E5))
 
 val Purple900 = Color(0xFF4A148C)
 val Purple500 = Color(0xFF9C27B0)
+val mapCircleFill = Color(0x8042A5F5)
```

Comment:
Are the maps dark-mode conscious?

Reply:
nope... I think the blue will work for both light and dark mode